ci-operator/config/redhat-developer/service-binding-operator/redhat-developer-service-binding-operator-master__4.6.yaml

```diff
@@ -1,12 +1,9 @@
-base_images:
-  os:
-    name: ubi
-    namespace: ocp
-    tag: "7"
-binary_build_commands: make build
 build_root:
   project_image:
-    dockerfile_path: openshift-ci/Dockerfile.tools
+    dockerfile_literal: "FROM registry.access.redhat.com/ubi8/ubi \nRUN yum install
+      -y --nodocs jq python3-virtualenv make git\nRUN curl -Lo /usr/bin/kubectl https://storage.googleapis.com/kubernetes-release/release/`curl
+      -s https://storage.googleapis.com/kubernetes-release/release/stable.txt`/bin/linux/amd64/kubectl
```

Comment:
not sure if we really need to install kubectl here, because in line 21 we somehow injecct clies into test container.

Reply:
also, you can use yaml multiline syntax for improved readability:

```
dockerfile_literal: |
  FROM registry.access.redhat.com/ubi8/ubi 
  RUN yum install -y -nodocs jq python3-virtualenv make git
```

Reply:
According to the [docs](https://docs.ci.openshift.org/docs/architecture/step-registry/#injecting-the-oc-cli), `oc` can be installed  via `cli: latest` directive... it does not mention `kubectl`

Reply:
> also, you can use yaml multiline syntax for improved readability:

Actually, I did but the `make ci-operator-config` that needs to be performed before commit changed that into what you see.
If I keep it "readable" and commit it like that, one of the PR checks (that do the same thing and compares the result with what is in the commit) fails.

Reply:
Anyway, let me try without explicitly downloding kubectl...

Reply:
Nope, `kubectl` is not installed via `cli: latest` - putting the download back.

Reply:
And suddnely, the multiline YAML works.

Reply:
but why do we need `kubectl` at all? I thought we use `oc` exclusively when running tests against openshift cluster? Where do we need pure kubectl?